### PR TITLE
Remove unnecessary function definition

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

**Comment by marcotcr** [comment]
I prefer to keep the kernel as a function, in case I want to refactor this later to include other kernels.
As for the second one - yes, that is a unnecessary one that should be removed.

File: lime/lime_tabular.py [hunk 1]
```diff
@@ -157,11 +157,10 @@ def __init__(self, training_data, training_labels=None, feature_names=None,
             kernel_width = np.sqrt(training_data.shape[1]) * .75
         kernel_width = float(kernel_width)
 
-        def kernel(d):
-            return np.sqrt(np.exp(-(d ** 2) / kernel_width ** 2))
-
         self.feature_selection = feature_selection
-        self.base = lime_base.LimeBase(kernel, verbose)
+        self.base = lime_base\
```
Comment (dhimmel) [inline lime/lime_tabular.py]: I'm not a fan of the readability here, how the command spans three lines.

Won't matter if this change is reverted.

Nudge @proinsias to continue with this PR.

**Comment by proinsias** [comment]
@marcotcr – updated.

**Comment by marcotcr** [comment]
Thanks

**Pull request merged**